cmd/provider/add.go

```diff
@@ -32,6 +35,11 @@ func NewAddCmd(flags *flags.GlobalFlags) *cobra.Command {
 	addCmd := &cobra.Command{
 		Use:   "add",
 		Short: "Adds a new provider to DevPod",
+		PreRun: func(ccmd *cobra.Command, args []string) {
```

Comment:
I think "ccmd" can be renamed to "cobraCmd", like we do in the root cmd, see: https://github.com/loft-sh/devpod/blob/main/cmd/root.go#L35.

```suggestion
		PreRun: func(cobraCmd *cobra.Command, args []string) {
```